src/Google.Cloud.Functions.Invoker/Extensions/FunctionsFrameworkWebHostBuilderExtensions.cs

```diff
@@ -41,35 +41,32 @@ public static class FunctionsFrameworkWebHostBuilderExtensions
             });
 
         /// <summary>
-        /// Configures the given web host builder to use the Functions Framework for typical use cases, performing
-        /// the following steps:
-        /// <list type="bullet">
-        /// <item><description>Configuration specific to the Functions Framework is added from the environment variables and command line arguments.</description></item>
-        /// <item><description>Existing logging providers are cleared, and the Functions Framework console logger is used (either writing a single line
-        /// per log entry, or writing JSON depending on the environment).</description></item>
-        /// <item><description>Kestrel is configured with the address and port configured from the environment and command line.</description></item>
-        /// <item><description>The function target is located within <paramref name="functionAssembly"/> based on configuration, or the function assembly
-        /// is scanned for a single function if the configuration does not specify a function target.
-        /// </description></item>
-        /// <item><description>Function startup classes (typically used to configure the services for dependency injection or to add middleware) are
-        /// added based on any <see cref="FunctionsStartupAttribute"/> values applied to the function assembly.
-        /// <item><description>The application is configured to use the Functions Framework request handler.</description></item>
-        /// </description></item>
-        /// </list>
+        /// Uses the functions startup classes specified by assembly attributes when configuring the host.
```

Comment:
nit: Maybe add a <see href.... /> for the attribute type.

Reply:
Done.